includes/Mapper/class-contactmapper.php

```diff
@@ -0,0 +1,168 @@
+<?php
```

Comment:
majority of these function does maps object to contact object. Why can't we have a single function that does this mapping?

i.e what is the main differnce betweeen get_omnisend_contact vs get_omnisend_update_contact ?

Reply:
these functions are triggered by different forms witch does not have all fields it is more convenient to process them separate.